src/meta/expressionEvaluator/functions/validatorNotGreaterThanForestOrMaxForest.ts

```diff
@@ -0,0 +1,34 @@
+import { ExpressionFunction } from '@openforis/arena-core/dist/expression/function'
+import { Numbers } from '@utils/numbers'
+import { Objects } from '@utils/objects'
+
+import { NodeValueValidation, NodeValueValidationMessage } from '@meta/assessment'
+import { validatorNotGreaterThanForest } from '@meta/expressionEvaluator/functions/validatorNotGreaterThanForest'
+
+import { Context } from '../context'
+
+export const validatorNotGreaterThanForestOrMaxForest: ExpressionFunction<Context> = {
+  name: 'validatorNotGreaterThanForestOrMaxForest',
+  minArity: 3,
+  executor: (context) => {
+    return (forestArea?: string, value?: string, maxForestArea?: string): NodeValueValidation => {
+      if (forestArea) return validatorNotGreaterThanForest.executor(context)(forestArea, value)
+
```

Comment:
here then you can call directly validatorNotGreaterThanMaxForest ?

Reply:
Here the idea is that if given forestArea, we know that we have an ODP for this year and directly call the correct validator for this.
If forestArea (=ODP value) is not given, we proceed to validate against max forest area

Reply:
sure, that's why I suggest to call validatorNotGreaterThanMaxForest after the if. would that work ?

Reply:
Definitely, fixed